modules/ROOT/pages/addition/index.adoc

```diff
@@ -350,15 +328,5 @@ image::agents.png[width=800]
 . Verify that the agent has a current status of `Online`. 
     ** If the agent is not currently online then go to where it is running and check the logs.
 . Hover over the newly added agent and click the cog icon to show agent configuration. Check configuration is as expected.
-. Navigate to the home page and wait for DBMS to appear, this can take a few minutes.
-. Select the _Alerts_ tab and make sure that there are no alerts for the new DBMS.
-
-=== Rename DBMS
-. Select System Topology 
-+
-. Open Context menu for DBMS node and select Edit
-+
-image::home-dbms-context-menu.png[width=800]
-+
-. Type new name
-+
\ No newline at end of file
+. Navigate to the home page (if this agent is the first to manage an instance in a DBMS, it may take a few minutes for the DBMS to appear)
```

Comment:
```suggestion
. Navigate to the home page (if this agent is the first to manage an instance in a DBMS, it may take a few minutes for the DBMS to appear).
```